src/services/user/userService.js

```diff
@@ -0,0 +1,22 @@
+export const UserService = {
+  getUser: async (userId) => {
```

Comment:
Ola boa tarde 😁
A sua tarefa esta relacionada a Listagem de Pets e o nome do método esta "getUser", seria interessante deixar mais semântico.

Reply:
O que seria mais interessante colocar? Porque nessa parte eu to pegando o usuário pra depois buscar o pet dele, imaginei que getUser seria ok

Reply:
Você ta buscando e retornando um Pet não um User, então getPet seria o ideal.

Reply:
Sim, é isso mesmo!

Reply:
Resolvido